src/empty-states/src/EmptyState.js

```diff
@@ -0,0 +1,184 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button } from '../../buttons'
+import { Pane } from '../../layers'
+import { majorScale, minorScale } from '../../scales'
+import { useTheme } from '../../theme'
+import { Link, Heading, Paragraph } from '../../typography'
+
+/* eslint-disable react/prop-types */
+
+const HorizontalOrientation = ({
+  anchorCta,
+  background,
+  description,
+  image,
```

Comment:
Should we name these props `icon` and `iconBgColor`?

Reply:
Yup good call!